core/types/block_ext.go

```diff
@@ -28,7 +28,7 @@ func (b *Block) setExtData(data []byte, recalc bool) {
 	b.extdata = &_data
 	copy(*b.extdata, data)
 	if recalc {
-		b.header.ExtDataHash = CalcExtDataHash(*b.extdata)
+		GetHeaderExtra(b.header).ExtDataHash = CalcExtDataHash(*b.extdata)
```

Comment:
nit: can we use a variable for extraHeader and then assign extDataHash?

Reply:
was this done?

Reply:
I did change and then revert it 🤣 
Are we in a tie??!
let's leave it as is, I am personally 50% shared between the two approaches, I don't think it matters much.